src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -77,7 +77,8 @@ public override Task WriteResponseBodyAsync([NotNull] OutputFormatterContext con
 
                 var type = GetSerializableType(context.DeclaredType, runtimeType);
                 var dataContractSerializer = CreateSerializer(type);
-                dataContractSerializer.WriteObject(xmlWriter, context.Object);
+                var responseObject = TryWrapSerializableErrorObject(context.Object);
```

Comment:
Don't use the term try, it indicates a different semantic.

I also don't see it in this PR, where is the code?